src/main/java/core/basesyntax/service/impl/ReportWriterServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class ReportWriterServiceImpl implements ReportWriterService {

```

Comment:
Seems like this service only writes data to file, so you can name it just FileWriterImpl and